fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/services/CredentialsService.java

```diff
@@ -25,11 +24,11 @@
 public interface CredentialsService {
 
     /**
-     * Get the credentials for the given servlet request
+     * Get the credentials for the given request
      *
-     * @param request the servlet request
+     * @param request the request
      * @return the JCR credentials for the given request
      */
-    Credentials getCredentials(final HttpServletRequest request);
+    Credentials getCredentials(final Object request);
```

Comment:
So our idea seems to be that anything could be used to generate JCR credentials, but secretly we know that we're going to factor out JCR credentials anyway, so who cares? Is that right?